google/cloud/spanner/samples/samples.cc

```diff
@@ -680,6 +703,16 @@ void RunAll() {
   std::cout << "\nRunning spanner_field_access_on_nested_struct sample\n";
   FieldAccessOnNestedStruct(client);
 
+  namespace s = google::cloud::spanner;
+  s::Client c2(s::MakeConnection(
+      s::Database(project_id, instance_id, database_id),
+      s::ConnectionOptions().enable_clog().enable_tracing("rpc")));
```

Comment:
Is this a "submit" thing or a "development testing" thing?

Reply:
Ooops, fixed, thanks!